CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to Dot
+
+Dot is a welcoming project, and we welcome the suggestions of all.
+However, certain types of contributions are more valuable than others.
```

Comment:
Should add a section below to start fleshing out the "more valuable than others" part of this.